Update installation instructions (#230)
precice/python-bindings: README.md

## Installing the package

Generally, it is recommended to work in a virtual environment when using the preCICE Python bindings. For this purpose, you might have to run `apt install python3-venv` first.

> Before running any of the commands below, please create a virtual environment at a location of your choice by running:

Reviewer: ```suggestion
Create a virtual environment in your working directory by running:
```
I would suggest sticking to imperative mood as we do in most of our documentation.

```bash
python3 -m venv .venv
```

> Using `.venv` is a common choice as path of your virtual environment. But you can use any path here. Next activate the virtual environment by running

Reviewer: ```suggestion
Using `.venv` is a common choice as path of your virtual environment. But you can use any path here. Activate the virtual environment by running
```
